traffic_ops/testing/api/vendor/github.com/kelseyhightower/envconfig/.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: go
```

Comment:
this dir needs to be added to `.rat-excludes` at the top level of tc or `rat` will complain...